test/10_before_test/cibuildwheel_test.py

```diff
@@ -0,0 +1,22 @@
+import os
+import utils
+
+def test():
```

Comment:
What about not adding yet another extra test folder, but just add it to `02_test` or `03_before_build`? Now that we're using `pytest` and are able to have multiple test functions, we can group the together much better?

@joerick ?

Reply:
Maybe yes. This is global question about test granulation. @joerick What you think about this?

Reply:
It would be nice to unify a little, but `10_before_test` is different than the others because a test within the project checks the existence of a file, rather than `setup.py`.

Reply:
Right, I see now :-)